src/grocker/helpers.py

```diff
@@ -76,7 +76,7 @@ def pip_conf(pip_conf_path=None):
 
     """
     if pip_conf_path is None or not os.path.exists(pip_conf_path):
-        with tempfile.NamedTemporaryFile('w', dir=os.path.expanduser('~/.cache')) as f:
+        with tempfile.NamedTemporaryFile('w', dir=os.path.expanduser('/tmp')) as f:
```

Comment:
```suggestion
        with tempfile.NamedTemporaryFile('w') as f:
```

No need to specify a location, `tempfile` can figure it out on its own.